conductor_api/src/conductor/base.rs

```diff
@@ -438,9 +449,12 @@ impl Conductor {
                 context_builder = context_builder.with_p2p_config(self.instance_p2p_config());
 
                 // Signal config:
-                self.setup_signals(signal_tx.clone());
-                context_builder = context_builder
-                    .with_signals(self.signal_tx.clone().expect("Signal sender not set up"));
+                let (sender, receiver) = unbounded();
```

Comment:
Why the switch to crossbeam channels?

Reply:
Because mpsc Receivers are mit Send, which would turn the whole Conductor struct not Send but is needed for admin interfaces and the lazy_static.

Reply:
That was confusing at first because I read "mit" as "with" and not "not" ;) Gotcha though